extensions/wikia/UserLogin/js/MarketingOptIn.js

```diff
@@ -12,21 +12,21 @@ define('usersignup.marketingOptIn', ['wikia.geo'], function (geo) {
 		var $optInField = wikiaForm.inputs.wpMarketingOptIn,
 			$optInGroup = wikiaForm.getInputGroup('wpMarketingOptIn'),
 			isEurope = geo.getContinentCode() === 'EU',
-			isCanada = geo.getCountryCode() === 'CA';
+			isCanada = geo.getCountryCode() === 'CA',
+			isJapan = geo.getCountryCode() === 'JP';
 
 		if (!$optInField.length || !$optInGroup.length) {
 			throw 'Wikia Form must contain a field called wpMarketingOptIn';
 		}
 
-		if (!isCanada) {
+		if (!isCanada && !isJapan) {
```

Comment:
Yep, I even started doing exactly that yesterday, then asked myself why, if we're just going to reformat it into a more mercury style later.